package.json

```diff
@@ -57,6 +57,10 @@
           "value": "de",
           "description": "Deutsch (de)"
         },
+        {
+          "value": "eo",
+          "description": "Esperanto (eo)"
```

Comment:
Confirmation: _Is Esperanto spelled as `Esperanto` in its own language?_

Reply:
yes it is